danger-kotlin/src/runnerMain/kotlin/com/danger/runner/cmd/Cmd.kt

```diff
@@ -0,0 +1,33 @@
+package com.danger.runner.cmd
+
+import platform.posix.system
+
+object Cmd : ICmd {
+    private const val DEFAULT_NAME = ""
+    private val DEFAULT_ARGS = arrayOf<String>()
```

Comment:
const?

Reply:
nope, not allowed for arrays as is not a primitive